Charliecloud.spec file fails to build — docutils version dependency

When building this rpm on a non-ubuntu OS, like Centos, we get the same error described by #119:

```
Warning, treated as error:
/home/charlie/rpmbuild/BUILD/charliecloud-0.9.2/doc-src/install.rst:127: ERROR: Error in "list-table" directive:
invalid option value: (option: "widths"; value: 'auto')
invalid literal for int() with base 10: 'auto'.

.. list-table::
   :widths: auto

   * - Distribution versions
     - proposed for *Buster* and *Stretch backports*
   * - Maintainers
     - Lucas Nussbaum (:code:`lucas@debian.org`)
       and Peter Wienemann (:code:`wienemann@physik.uni-bonn.de`)
   * - Bug reports to
     - Charliecloud's GitHub issue tracker
   * - Packaging source code
     - in Charliecloud: :code:`packaging/debian`

make: *** [html] Error 1
make: Leaving directory `/home/charlie/rpmbuild/BUILD/charliecloud-0.9.2/doc-src'
error: Bad exit status from /var/tmp/rpm-tmp.kdfiZu (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.kdfiZu (%build)
```

Our spec file has the following build requirements:

```spec
11 BuildRequires: python python-sphinx python-sphinx_rtd_theme rsync
```

--- a/packaging/redhat/charliecloud.spec
+++ b/packaging/redhat/charliecloud.spec
@@ -14,14 +14,6 @@ BuildRequires: python python-sphinx python-sphinx_rtd_theme rsync
 Charliecloud provides user-defined software stacks (UDSS) for
 high-performance computing (HPC) centers.
 
-%package doc
-Summary: Documentation and examples for %{name}
-Group: System Environment/Base
-
-%description doc
-This package contains documentation and examples
-for the %{name} package.
-
 %prep
 %setup -q
 # Required for CentOS 7 and older, which don't know of Docker lexer yet.
@@ -29,8 +21,6 @@ for the %{name} package.
 
 %build
 %{__make} %{?mflags}
-%{__make} -C doc-src %{?mflags}
-mv doc html
 
 %install
 LIBEXEC_POSTFIX=$(echo %{_libexecdir} | sed 's#^/usr/##')
@@ -58,8 +48,5 @@ rm -rf $RPM_BUILD_ROOT
 
 # Binaries
 %{_bindir}/ch-*
-
-%files doc
-%doc html
 
 %changelog

